granary/tests/test_reddit.py

```diff
@@ -191,16 +209,28 @@ def setUp(self):
     self.reddit = reddit.Reddit('token-here')
 
   def test_missing_user_to_actor(self):
-    self.assert_equals(MISSING_ACTOR, self.reddit.praw_to_actor(FakeMissingRedditor()))
+    self.assert_equals(MISSING_OBJECT, self.reddit.praw_to_actor(FakeMissingRedditor()))
```

Comment:
might be time to switch to `util.Struct` inline in tests instead of proliferating these one-off classes? eg here, `s/FakeMissingRedditor()/util.Struct()`...